designer/client/src/components/graph/CursorMask.ts

```diff
@@ -1,31 +1,17 @@
-import { css } from "@emotion/css";
 import { CSSProperties } from "react";
 
 export class CursorMask {
-    private static instance: CursorMask;
     private readonly cursorMask: HTMLElement;
 
-    constructor() {
-        this.cursorMask = document.createElement("DIV");
-        this.cursorMask.className = css({
-            cursor: "not-allowed",
-            position: "absolute",
-            zIndex: 100000,
-            top: 0,
-            bottom: 0,
-            left: 0,
-            right: 0,
-        });
+    constructor(element: HTMLElement) {
+        this.cursorMask = element;
```

Comment:
It fixes the issue with pan on firefox, not sure why, new div element creation blocked pan effect 🤷‍♂️